### ISSUE TITLE
historian.ts reference to window causing problems for 1.4.7

### ISSUE THREAD

**Issue by andykellr** [airbrake/airbrake-js#469]
I replaced my specific build paths with `___` since they're aren't relevant.

Forcing version 1.4.6 makes this go away so this seems like a new issue.

```
ReferenceError: window is not defined
    at Object../src/historian.ts (___/node_modules/airbrake-js/dist/webpack:/airbrakeJs.Client/src/historian.ts:401:1)
    at __webpack_require__ (___/node_modules/airbrake-js/dist/webpack:/airbrakeJs.Client/webpack/bootstrap:19:1)
    at Object../src/client.ts (___/node_modules/airbrake-js/dist/webpack:/airbrakeJs.Client/src/client.ts:24:1)
    at __webpack_require__ (___/node_modules/airbrake-js/dist/webpack:/airbrakeJs.Client/webpack/bootstrap:19:1)
    at Object.0 (___/node_modules/airbrake-js/dist/client.js:2678:18)
    at __webpack_require__ (___/node_modules/airbrake-js/dist/webpack:/airbrakeJs.Client/webpack/bootstrap:19:1)
    at ./node_modules/error-stack-parser/error-stack-parser.js.define (___/node_modules/airbrake-js/dist/webpack:/airbrakeJs.Client/webpack/bootstrap:83:1)
    at ___/node_modules/airbrake-js/dist/client.js:98:10
    at webpackUniversalModuleDefinition (___/node_modules/airbrake-js/dist/webpack:/airbrakeJs.Client/webpack/universalModuleDefinition:3:1)
    at Object.<anonymous> (___/node_modules/airbrake-js/dist/webpack:/airbrakeJs.Client/webpack/universalModuleDefinition:10:2)
    at Module._compile (module.js:652:30)
```

**Comment by vmihailenco** [comment]
Can you please describe the problem with 1.4.7 in more details? Is there a backtrace?

**Comment by vmihailenco** [comment]
Hopefully fixed in v1.4.8
